packages/preview-middleware-client/test/unit/adp/controllers/AddFragment.controller.test.ts

```diff
@@ -168,6 +189,49 @@ describe('AddFragment', () => {
         });
     });
 
+    describe('onSpecialIndexIconPress', () =>{
+
+        afterEach(() => {
+            jest.restoreAllMocks();
+        });
+
+        test('on icon clicked', () =>{
+
+            const addFragment = new AddFragment(
+                'adp.extension.controllers.AddFragment',
+                {} as unknown as UI5Element,
+                {} as unknown as RuntimeAuthoring,
+            );
+
+            addFragment['runtimeControl'] = {
+                getMetadata: jest.fn().mockReturnValue({
+                    getName: jest.fn().mockReturnValue('Toolbar')
+                })
+            } as unknown as ManagedObject;
+
+            const oEvent = {
+                getSource: jest.fn().mockReturnValue({})
+            };
+
+            const icon = oEvent.getSource();
+            
+
+            const setPropertySpy = jest.fn();
+            const openBySpy = jest.fn();
+
+            addFragment['popover'] = {
+                getContent: jest.fn().mockReturnValue([{setProperty: setPropertySpy}]),
+                openBy: openBySpy
+            } as unknown as Popover;
+
+            addFragment.onSpecialIndexIconPress(oEvent as unknown as Event)
```

Comment:
Remove Hungarian notation

Reply:
resolved in [e94a899](https://github.com/SAP/open-ux-tools/pull/1477/commits/e94a89919304cadb80bf98ae1eb4ece1069cc8cb)